.circleci/config.yml

```diff
@@ -214,10 +214,16 @@ jobs:
             ./fil-proofs-tooling/scripts/benchy-remote.sh "${BENCHMARK_SERVER_SSH_USERNAME}@${BENCHMARK_SERVER_IP_ADDR}" zigzag --size=$((1024*1024)) > zigzag-benchmarks.json
             cat zigzag-benchmarks.json
           no_output_timeout: 60m
+      - run:
+          name: Run Rational PoST benchmarks using a 1MiB sector
+          command: |
+            ./fil-proofs-tooling/scripts/benchy-remote.sh "${BENCHMARK_SERVER_SSH_USERNAME}@${BENCHMARK_SERVER_IP_ADDR}" rational-post --size=1024 > rational-post-benchmarks.json
```

Comment:
QUESTION: What happens if you run this thing with `--size=((1024 * 1024))` instead of `--size=1024`? Does it run successfully?